src/pkg/utils/utils.go

```diff
@@ -109,6 +109,7 @@ var (
 		"XDG_SESSION_ID",
 		"XDG_SESSION_TYPE",
 		"XDG_VTNR",
+		"ZDOTDIR",
```

Comment:
It will be good to add some tests for `ZDOTDIR` to `test/system/220-environment-variables.bats`.

Reply:
I have added some tests. I had some difficulties running them locally, but the bot seems to be happy.

![image](https://github.com/containers/toolbox/assets/138280734/e6a7fa8f-b887-40fe-a5d1-97659f4770ed)

[system-test-fedora-39/console](https://softwarefactory-project.io/zuul/t/local/build/78255f92bc574d0bb7858fe891dbc437/console)